local_store.go

```diff
@@ -124,13 +124,33 @@ func (f *fileSystemStore) stagedDir() string {
 	return filepath.Join(f.dir, "staged")
 }
 
+func (f *fileSystemStore) keysDir() string {
+	return filepath.Join(f.dir, "keys")
+}
+
 func (f *fileSystemStore) GetMeta() (map[string]json.RawMessage, error) {
 	meta := make(map[string]json.RawMessage)
 	var err error
 	notExists := func(path string) bool {
 		_, err := os.Stat(path)
 		return os.IsNotExist(err)
 	}
+	if !notExists(f.stagedDir()) {
+		dir, err := ioutil.ReadDir(f.keysDir())
+		if err != nil {
+			return meta, err
+		}
+		//adding all delegated role names into top level manifests
+		//so that it can be initiated in the system
+		for _, fileInfo := range dir {
+			nameJSON := fileInfo.Name()
+			if nameJSON != "root.json" && nameJSON != "targets.json" && nameJSON != "snapshot.json" && nameJSON != "timestamp.json" && strings.HasSuffix(nameJSON, ".json") {
```

Comment:
These are a lot statements. I wonder if it would be possible to make this easier to read. Maybe more than one if statement?.. What do you think? :)